lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -29,18 +29,21 @@
 else:
     # Read a .pem file from AWS Parameter Store.
 
-    ssm_client = boto3.client("ssm")
+    # AWS_REGION is set during unit testing.  In a deployment environment,
+    # region_name=None will result in boto using the "default session".
+    #   https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/boto3.html?highlight=boto3.client#boto3.client
+#    ssm_client = boto3.client("ssm", region_name=os.getenv("AWS_REGION"))
 
-    response = ssm_client.get_parameter(
-        Name=f"/{NOTIFY_ENVIRONMENT}/notification-api/profile-integration-pem",
-        WithDecryption=True
-    )
+#    response = ssm_client.get_parameter(
+#        Name=f"/{NOTIFY_ENVIRONMENT}/notification-api/profile-integration-pem",
+#        WithDecryption=True
+#    )
 
-    PEM = response.get("Parameter", {}).get("Value", None)
+#    PEM = response.get("Parameter", {}).get("Value", None)
```

Comment:
Commented-out because I don't want to make calls to AWS during testing.  I probably need a context override for the test.